### PR TITLE
Update CHANGES.md for v1.0.17

### PR DESCRIPTION
**Pull request by stealthcode**

### CODE REVIEW

**Comment by stealthcode** [comment]
Is there anything else not yet merged that is urgent or high priority that we should consider before cutting this release?

**Comment by artem-zinnatullin** [comment]
Well, I hoped to add some `rx.Single` operators, but looks like I won't be able to do it this week. Can we expect 1.1.1 this month?

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 # RxJava Releases #
 
+### Version 1.0.17 – December 1 2015 ([Maven Central](http://search.maven.org/#artifactdetails%7Cio.reactivex%7Crxjava%7C1.0.17%7C)) ###
+
+* [Pull 3150] (https://github.com/ReactiveX/RxJava/pull/3150) Window operators now support backpressure in the inner observable
+* [Pull 3535] (https://github.com/ReactiveX/RxJava/pull/3535) Don't swallow fatal errors in OperatorZipIterable
+* [Pull 3528] (https://github.com/ReactiveX/RxJava/pull/3528) Avoid to call next when Iterator is drained
+* [Pull 3436] (https://github.com/ReactiveX/RxJava/pull/3436) Add action != null check in OperatorFinally
+* [Pull 3513] (https://github.com/ReactiveX/RxJava/pull/3513) Add shorter RxJavaPlugin class lookup approach
```
Comment (artem-zinnatullin) [inline CHANGES.md]: Optional: `(useful for Android)`

Comment (stealthcode) [inline CHANGES.md]: A qualifier or explanation is not necessary in the release notes since there is a link to the PR.

**Comment by stealthcode** [comment]
@artem-zinnatullin there isn't a release schedule so I really couldn't tell you.

**Comment by artem-zinnatullin** [comment]
okay, LGTM :+1:

**Comment by akarnokd** [comment]
#3491

**Comment by stevegury** [comment]
:+1:

**Comment by davidmoten** [comment]
#3491 from me too. We really need this bug fix in.

**Comment by stealthcode** [comment]
I'm looking at #3491

**Comment by stealthcode** [comment]
I've gone ahead and merged #3491. Sorry for the delay.

**Comment by akarnokd** [comment]
:+1:

**Pull request merged**